.github/PULL_REQUEST_TEMPLATE/add_repo.md

```diff
@@ -0,0 +1,27 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+* Consider creating this PR as draft: https://github.com/kubevirt/kubevirt/blob/main/CONTRIBUTING.md#consider-opening-your-pull-request-as-draft
+-->
+
+**What this PR does / why we need it**:
+
+This PR adds the repository `repo_name` to the KubeVirt organization.
+
+_Checklist:_
+
+- [ ] repo is a good fit for the org
+- [ ] maintainers are ok with adding the repo
+- [ ] if CI support is planned, CI capacity will not get exhausted
+- [ ] licensing model fits the org license model
+
+CI support is planned for:
```

Comment:
Maybe add GitHub Actions? Or at least add a note that they could be used too in special cases?